lib/resources/aide_conf.rb

```diff
@@ -1,11 +1,11 @@
 # encoding: utf-8
-# author: Jen Burns, burnsjennifere@gmail.com
```

Comment:
Have we agreed somewhere that we remove the author headers?

Reply:
Yes, github accurately keeps history of authors. This is not the main focus of this PR but is being included for now.

Reply:
Lets try to keep a PR focused on one thin in the future

Reply:
No problem, our Team Working Agreement states that we should remove these when the file is touched so thats what I did.